File: server/config/brand.js
const BRAND = Object.freeze({
  name: 'TruFusionLabs',
  appUrl: 'https://www.trufusionlabs.com',
  apexUrl: 'https://trufusionlabs.com',
  apiUrl: 'https://api.trufusionlabs.com',
  shopUrl: 'https://shop.trufusionlabs.com',
  portUrl: 'https://port.trufusionlabs.com',
  supportEmail: 'support@trufusionlabs.com',
  legacySupportEmail: 'support@peppro.net',
  logoPath: 'public/TruFusionLabs_PhysiciansPortal.png',
});

const LEGACY_BRAND = Object.freeze({
  name: 'PepPro',
  orderTable: 'peppro_orders',
  orderMetaPrefix: 'peppro',
  supportEmail: 'support@peppro.net',
});

const legacyMetaKey = (key) => {
  const normalized = String(key || '');
  return normalized.startsWith('trufusion')
    ? `peppro${normalized.slice('trufusion'.length)}`
    : null;
};

const withLegacyMetaKeys = (keys) => {
  const list = Array.isArray(keys) ? keys : [keys];
  const expanded = [];
  list.forEach((key) => {
    if (key && !expanded.includes(key)) expanded.push(key);
    const legacy = legacyMetaKey(key);
    if (legacy && !expanded.includes(legacy)) expanded.push(legacy);
  });
  return expanded;
};

module.exports = {
  BRAND,
  LEGACY_BRAND,
  legacyMetaKey,
  withLegacyMetaKeys,
};

File: python_backend/brand.py
BRAND = {
    "name": "TrufusionLabs",
    "app_url": "https://www.trufusionlabs.com",
    "apex_url": "https://trufusionlabs.com",
    "api_url": "https://api.trufusionlabs.com",
    "shop_url": "https://shop.trufusionlabs.com",
    "port_url": "https://port.trufusionlabs.com",
    "support_email": "support@trufusionlabs.com",
    "legacy_support_email": "support@trufusionlabs.com",
    "logo_path": "public/turfusionlabsphysiciansportal.png",
}

LEGACY_BRAND = {
    "name": "PepPro",
    "order_table": "peppro_orders",
    "order_meta_prefix": "peppro",
    "support_email": "support@trufusionlabs.com",
    "media_auth_cookie_name": "peppro_media_token",
}


def legacy_meta_key(key: str) -> str | None:
    if not key.startswith("trufusion"):
        return None
    return f"peppro{key[len('trufusion'):]}"


def with_legacy_meta_keys(*keys: str) -> list[str]:
    expanded: list[str] = []
    for key in keys:
        if key and key not in expanded:
            expanded.append(key)
        legacy = legacy_meta_key(key or "")
        if legacy and legacy not in expanded:
            expanded.append(legacy)
    return expanded
